lib/buffer.js

```diff
@@ -213,7 +214,7 @@ function allocate(size) {
     // Even though this is checked above, the conditional is a safety net and
```

Comment:
Side note: this comment is irrelevant now, probably since dd67608, I overlooked it.
/cc @trevnorris